students/seongsu/westagram/users/views.py

```diff
@@ -54,11 +54,13 @@ def post(self, request):
             email    = user_data['email']
             password = user_data['password']
             user = User.objects.get(email = email)
```

Comment:
```suggestion
            email    = user_data['email']
            password = user_data['password']
            user     = User.objects.get(email = email)
```
convention만 조금 맞춰주시면 완벽할 것 같습니다 성수님!

Reply:
수정했습니다~